server/data_provider_canvas_esb.rb

```diff
@@ -76,22 +85,29 @@ def initConfigureCanvasESBProvider(config_hash)
 
   def canvasAPICalendarEventsURL(uniqname, canvas_courses)
 
-    ## calculate a 15 day range around now for the assignment search (last week, today, next week)
+    ## calculate a multi day range around now for the assignment search (last week, today, next week).
+    ## Can override defaults (above) in the studentdashboard.yml file.
```

Comment:
Is there a studentdashboard.yml file? I only found studentdashboard-LOCALHOST.yml file. Or is studentdashboard.yml a security/configuration file only exist on deployment?